crates/query-engine/sql/src/sql/convert.rs

```diff
@@ -166,12 +166,12 @@ impl From {
                 sql.append_syntax(" AS ");
                 alias.to_sql(sql);
             }
-            From::JsonToRecordset {
+            From::JsonbToRecordset {
```

Comment:
We use `jsonb` instead of `json` because json-valued query parameters appear to be bound as `jsonb`, which means we avoid having to cast (see the updated test snapshots for examples)